.github/workflows/update_issues.yml

```diff
@@ -0,0 +1,50 @@
+name: update issues
+on:
+  schedule:
+    - cron: "0 1 * * *"
+
+jobs:
+  update_issues:
+    runs-on: ubuntu-latest
+
+    steps:
+      # ----------------------------------------------------------------------------------
+      # Setup
+      # ----------------------------------------------------------------------------------
+      - uses: actions/checkout@v2
+
+      - uses: ./.github/actions/setup-python
+      - uses: ./.github/actions/install-dependencies
+
+      - name: Set up git config
```

Comment:
I think this setting up of git config is a duplicate of text from another workflow, so sooner or later, we should eliminate the duplicate, and move the text to a new folder in `./.github/actions/` alongside `setup-python`...

Reply:
I have extracted the git config setup step to a separate action, but did not touch the original source of the code.